SimPEG/EM/Base.py

```diff
@@ -303,41 +331,26 @@ def MeSigma(self):
             self._MeSigma = self.mesh.getEdgeInnerProduct(self.sigma)
         return self._MeSigma
 
-    @property
```

Comment:
similar comment. we need to make sure that we are updating this in the inversion.

Reply:
This does get cleared on updates to sigma or rho - since `_MeSigmaDeriv` is in the list of `_clear_on_sigma_update`, it will be deleted if there is an update to sigma in the inversion